CodeNow/runnable-angular
# SAN-2410 - File Upload Save

Fixed bug where we weren't adding a loading promise when a file was being uploaded!

To test:
1. Go to a non-repo container, click edit
2. Add a file
3. Hit save
4. Wait for save to finish
5. Click edit again and verify the file is still there
